src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -369,6 +400,13 @@ public function testThrowExceptionIfFormatDoesNotContainYearMonthAndDay()
      */
     public function testThrowExceptionIfFormatMissesYearMonthAndDayWithSingleTextWidget()
     {
+        if (method_exists($this, 'expectException')) {
```

Comment:
let's use the `@expectedException` annotation instead

Reply:
done